src/plugins/intel_npu/src/al/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ endif()
 target_link_libraries(${TARGET_NAME}
     PUBLIC
         openvino::npu_logger_utils
+        openvino::npu_zero_utils
```

Comment:
In this case, and targets depend on al will have new deps of ze loader, right? I think compiler lib may forced to be linked with ze_loader now, and offline compilation may be corrupted。

Reply:
Indeed. it will depend on the ze_loader targets. But not sure that I understand correctly why it may be corrupted.

Reply:
If we use CIP to do compilation without NPU device, not sure if it can find ze_loader

Reply:
even today we need ze_loader to be there.